.github/workflows/release.yml

```diff
@@ -1,43 +1,63 @@
 name: Release
 
 # Taken from: https://github.com/changesets/action#with-publishing
-# TODO: get/set GITHUB_TOKEN and NPM_TOKEN
 
 on:
   push:
     branches:
-      - DISABLED_main
+      - main
 
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
   release:
-    name: Release
+    name: Changelog PR or Release
+    if: ${{ github.repository_owner == 'kadena-community' }}
     runs-on: ubuntu-latest
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+
     steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
         with:
           show-progress: false
 
       - name: Setup Node.js 18
         uses: actions/setup-node@v3
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
+      - uses: actions/setup-node@v3
         with:
-          node-version: 18
+          # Node.js v18 gives error in changesets/action@v1 (can't find dist/index.js)
+          node-version: 16
+          cache: 'pnpm'
 
-      - name: Install Dependencies
+      - name: Install dependencies
         run: pnpm install
 
-      - name: Create Release Pull Request or Publish to npm
+      - name: Build, lint, test all packages
+        run: pnpm turbo build lint test
+
+      - name: Create Release PR or Publish
         id: changesets
         uses: changesets/action@v1
         with:
-          publish: pnpm publish-packages
+          # Note: pnpm install after versioning is necessary to refresh lockfile
+          version: pnpm run version
+          publish: pnpm run publish
```

Comment:
`publish` command is default npm publish command or it triggers the publish command(`changeset publish`) from `package.json`

Reply:
its juts an alias for "changeset publish", check the root package.json